### PR TITLE
Fetch and render api

### PR DESCRIPTION
**Pull request by RicardoBachmann**
I tried to add the catch statment in case for offline API but it faild

### CODE REVIEW

File: src/components/ParticipantCard.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {render, screen} from '@testing-library/react';
+
+import ParticipantCard from './ParticipantCard.js';
+
+describe('ParticipantCardn', () => {
+  it('should render the data when api responds', () => {
+    render(<ParticipantCard />);
+    const participantCard =screen.getByRole('list');
+    expect(participantCard).toBeInTheDocument();
+    document.querySelector('section');
```
Comment (stefankorder) [inline src/components/ParticipantCard.spec.js:10]: querySelector is not needed at this point.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Just one mistake.

File: src/components/ParticipantCard.spec.js (hunk 1 above)
Comment (RicardoBachmann) [reply src/components/ParticipantCard.spec.js:10]: okay thank u! I delete the code line.

**Review by stefankorder — APPROVED** [review]
👍🏻

File: src/App.js [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
+        <ParticipantCard
+          key={features.id}
+          name={features.properties.business_name}
+          business_type={features.properties.business_type}
+          address={features.properties.address}
+          live_date={features.properties.live_date}
+          precinct={features.properties.precinct}
+          error={error}
+          loading={loading}
+          participantData={participantData}
```
Comment (JessicaLoers) [inline src/App.js:47]: These props are not used "anymore" in the  `ParticipantCard` component. Please remove 🙂
```
error={error}
loading={loading}
participantData={participantData}
```

File: src/components/ParticipantCard.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+export default function ParticipantCard({error, loading, name, business_type, address, live_date, precinct}) {
```
Comment (JessicaLoers) [inline src/components/ParticipantCard.js:3]: These props aren't used "anymore" in the component. Please remove 🙂

`{error, loading}`

**Review by JessicaLoers — COMMENTED** [review]
**Almost done**, just minor comments. 🙃

**Review by JessicaLoers — APPROVED** [review]
It's a wrap!
**Well done, Ricardo** 🚀

**Pull request merged**